### PR TITLE
fix: logical error in the conditional checks for model classes

### PR DESCRIPTION
**Pull request by ShaneOss**
Fix logical error in the conditional checks for model classes. The `elif` statement that looks for "lightgbm.sklearn" or "xgb" in the model class string is now broken into two separate conditions because the old condition would always evaluate to `True` due to the non-empty string "xgb".

### CODE REVIEW

**Review by xmatthias — DISMISSED** [review]
Purely looking at it i know this is the right approach.
I'd want @robcaulk to give his final say though - as i see him as code-owner for this part of the code

LGTM :+1:

File: freqtrade/freqai/utils.py
Comment (robcaulk) [inline freqtrade/freqai/utils.py]: If we are going to improve the grammar, let's fix it fully :)

"Model type does not support generating feature importances"

Comment (ShaneOss) [reply freqtrade/freqai/utils.py]: Thanks Rob. Yeah I noticed that after creating the PR. Cheers!

**Review by xmatthias — APPROVED** [review]

**Pull request merged**
